product_docs/docs/pem/8/registering_agent.mdx

```diff
@@ -9,46 +9,66 @@ redirects:
 - /pem/latest/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/09_registering_a_pem_agent/ 
 ---
 
-You must register each PEM agent installed on a separate host with the PEM server. (The PEM agent is different from the PEM server host.) The registration process provides the PEM server with the information it needs to communicate with the agent. The PEM agent graphical installer for Windows supports self-registration for the agent. On a Linux host, you must use the pemworker utility to register the agent.
+Before a PEM agent can be used, you must register it with a PEM server. 
+**PEM agents installed by the PEM server package are registered automatically during server configuration.** For all other agents you must follow the instructions below.
 
-The PEM agent package installer places the PEM agent in the `/usr/edb/pem/agent/bin` directory. To register an agent, include the `--register-agent` keywords along with registration details when invoking the pemworker utility:
+!!! Note
+    After upgrading the PEM agent, you need to restart it. You do not need to register it again. 
+
+## How to register PEM agents
+On Linux and Windows hosts, the PEM agent package includes command line utility called 'pemworker' which you can use perform management tasks, including registering the PEM agent, as described [below](#registering-a-pem-agent-using-the-pemworker-utility).
+
+On Windows, the PEM agent graphical installer allows you to register the agent when installing it. This is a convenience option and does not support all the options provided by pemworker. 
+If you do not want the installer to register the agent, uncheck the 'register now' checkbox. For more details, refer to the [installation instructions](/pem/latest/installing/windows/).
+
+## Registering a PEM agent using the pemworker utility
+The PEM agent package installer places the pemworker utility in the `/usr/edb/pem/agent/bin` directory on Linux and `C:\Program Files\edb\pem\agent-x64`bin` on Windows. 
+To register an agent, invoke the utility as shown below and add the relevant options from the table as needed. Follow each option with a corresponding value.
```

Comment:
```suggestion
To register an agent, invoke the utility below and add the relevant options from the table as needed. Follow each option with a corresponding value.
```

Reply:
The utility isn't below, an invocation of it is. Therefore, I think the original is more precise.